streams/src/test/java/org/apache/kafka/streams/integration/KafkaStreamsCloseOptionsIntegrationTest.java

```diff
@@ -90,6 +90,10 @@ public static void startCluster() throws IOException {
 
     @AfterAll
     public static void closeCluster() {
+        if (adminClient != null) {
+            adminClient.close();
```

Comment:
what about doing this quietly (reusing `Utils.closeQuietly`)

Reply:
That were be more concise, thanks for the suggestion 😄